parbase/FairParAsciiFileIo.h

```diff
@@ -23,20 +23,35 @@ class FairParAsciiFileIo : public FairParIo
   public:
     FairParAsciiFileIo();
 
-    // default destructor closes an open file and deletes list of I/Os
+    /**
+     * default destructor closes an open file and deletes list of I/Os
+     */
     ~FairParAsciiFileIo();
 
-    // opens file
-    // if a file is already open, this file will be closed
-    // activates detector I/Os
+    /**
+     * \brief Opens file
+     * If a file is already open, this file will be closed,
+     * activates detector I/Os.
+     */
     Bool_t open(const Text_t* fname, const Text_t* status = "in");
 
-    // concatenate files whose names are stored in the TList
-    // TList holds list od TObjStrings
-    // create file all.par in local working directory
-    // calls open to open the generated file all.par
+    /**
+     * \brief Opens TList of files
+     * Concatenates files whose names are stored in the TList,
+     * TList holds list of TObjStrings.
+     * Calls MergeFiles with all_$pid.par in local working directory,
+     * calls open to open the generated file all_$pid.par.
+     */
     Bool_t open(const TList* fnamelist, const Text_t* status = "in");
```

Comment:
Awesome to see Doxygen style comments here!

BTW, You can look at the rendered result here:

http://lxci001.gsi.de/ci/tmp/jenkins-FairRootGroup-FairRoot-PR-1324-2/d1/d95/classFairParAsciiFileIo.html#a22b42eeecee09fc6eb6f19ee591a4f35

Reply:
Thanks, was looking for that!

Reply:
Updated rendering: http://lxci001.gsi.de/ci/tmp/jenkins-FairRootGroup-FairRoot-PR-1324-3/d1/d95/classFairParAsciiFileIo.html#ade967fe3135a3457c9c2cc3eef42de3c